gnovm/stdlibs/std/native.gno

```diff
@@ -14,41 +14,41 @@ func GetChainID() string     // injected
 func GetChainDomain() string // injected
 func GetHeight() int64       // injected
```

Comment:
Can you change it in the above functions as well?

Reply:
Refactored in https://github.com/gnolang/gno/pull/3374/commits/5c6abafb639ee30bab242b8af27dcd110f014209

Reply:
Let's go for `ChainHeight` instead; I think it's reasonable considering we also have `ChainDomain` now.

Reply:
it's related to these codes also: 
```
func (h *Header) GetChainID() string { return h.ChainID }
func (h *Header) GetHeight() int64   { return h.Height }
func (h *Header) GetTime() time.Time { return h.Time }
```

Should we change them to ChainHeight, ChainTime ? 
And also, for the case of ChainID, there is a conflict in method name and attr name.

IMHO, all others refactoring are fine but this one is quite tricky. Should we do this ? if yes then  should we address these changes in another PR instead ?

Reply:
The name `ChainHeight` is likely temporary as we will then split up the packages following this pr, where IIRC this should likely be something like `chain.Height`.

We don't necessarily have to make these match the Header type.

IMO, make this `ChainHeight` now; we might rename it again when we split up the `std` package.

Reply:
Done in https://github.com/gnolang/gno/pull/3374/commits/b994dde0671338951ea9e3e7e918f5f65312e989 and https://github.com/gnolang/gno/pull/3374/commits/9e7881780ba6419d753272431ef54d37ef37cfa5

Reply:
I've made the necessary changes :) @thehowl